create_requirement_images.py

```diff
@@ -40,20 +42,26 @@
 bundle_data = json.load(f)
 f.close()
 
-font = ImageFont.truetype("Roboto-Regular.ttf", 24)
-right_triangle = Image.open("img/right_triangle.png")
-down_triangle = Image.open("img/down_triangle.png")
 
-folder_icon = Image.open("img/folder.png")
-folder_hidden_icon = Image.open("img/folder_hidden.png")
-file_icon = Image.open("img/file.png")
-file_hidden_icon = Image.open("img/file_hidden.png")
-file_empty_icon = Image.open("img/file_empty.png")
-file_empty_hidden_icon = Image.open("img/file_empty_hidden.png")
+def asset_path(x):
```

Comment:
Very minor, but would be good if we can use `asset_name` or `filename` or something more descriptive than `x` for the parameter here.